impl/thread.cxx

```diff
@@ -25,5 +40,49 @@ std::thread::native_handle_type currentThread()
 	throw std::runtime_error("not implemented");
 #endif
 }
+
+uint32_t currentCpu()
```

Comment:
For consistency, we'd name this `currentCPU` as 'CPU' is an initialism, not a word